packages/diagrams/backend/sirius-components-diagrams-layout/src/test/java/org/eclipse/sirius/components/diagrams/layout/incremental/DiagramLayoutTests.java

```diff
@@ -127,21 +127,21 @@ public Optional<IRepresentationDescription> findById(IEditingContext editingCont
     public void testRemoveEdgeDoesNotAffectUnrelated() {
 
         Diagram diagram = TestLayoutDiagramBuilder.diagram(DIAGRAM_ROOT_ID)
-            .nodes()
+                .nodes()
                 .rectangleNode(FIRST_TARGET_OBJECT_ID).at(10, 10).of(20, 20).and()
                 .rectangleNode(SECOND_TARGET_OBJECT_ID).at(50, 10).of(20, 20).and()
                 .rectangleNode(THIRD_TARGET_OBJECT_ID).at(50, 40).of(20, 20).and()
                 .and()
-            .edge("one")
+                .edge("one")
                 .from(FIRST_TARGET_OBJECT_ID).at(0.75, 0.5)
                 .to(SECOND_TARGET_OBJECT_ID).at(0.25, 0.5)
                 .and()
-            .edge("two")
+                .edge("two")
                 .from(FIRST_TARGET_OBJECT_ID).at(0.5, 0.75)
                 .to(THIRD_TARGET_OBJECT_ID).at(0.25, 0.5)
                 .goingThrough(20, 50)
                 .and()
-            .build();
+                .build();
```

Comment:
I think the previous formatting was on purpose, to clearly identify "sub blocks" corresponding to the different elements being built.

Reply:
I keep the old formatting